src/grandorgue/sound/GOSoundReverb.cpp

```diff
@@ -81,13 +81,15 @@ void GOSoundReverb::Setup(GOConfig &settings) {
       data[i] *= gain;
     if (len >= offset + settings.ReverbLen() && settings.ReverbLen())
       len = offset + settings.ReverbLen();
+    /*
```

Comment:
Are those logs comment wanted ?

Reply:
I added this log for debugging #983 and I forgot to remove it before some previous PR.

Because I'm still working on this issue, I still need this log, so I'd not remove it.

So now I'm commenting out this logging.